Automattic/simplenote-android
# Fix editor's background color in landscape mode with dark theme

Fixes #1410.

### Fix

When dark mode is enabled, rotating to landscape orientation while editing should retain dark mode. We did this by disabling full-screen keyboard (`android:imeOptions="flagNoExtractUi"`) and hiding toolbar for landscape mode in phones. 

**Before:**

https://user-images.githubusercontent.com/42454301/124658653-05cea980-de72-11eb-8f8b-0a9fec888ba7.mp4

**After:**

https://user-images.githubusercontent.com/195721/130622851-e273e4b6-6673-4e57-ab55-e2bd2e293b99.mp4

### Test

1. Go to settings and enable dark mode
2. Tap on a new note or a note from the notes list
3. Start typing on the note
4. Rotate the phone to landscape mode
5. The tabs and toolbar should hide and the background of the text editor should remain dark
6. Rotate the phone again
7. The tabs and toolbar should be shown again
8. Rotate the phone to landscape mode
9. The tabs and toolbar should hide and the background of the text editor should remain dark
10. Hide the keyboard
11. The tabs and toolbar should be shown again
12. Disable markdown
13. Rotate the phone to landscape mode
14. Start typing on the note
15. The toolbar should hide and the background of the text editor should remain dark
16. Hide the keyboard
17. The toolbar (not the tabs) should be shown again

### Review

Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release

`RELEASE-NOTES.txt` was updated in 31fedccb72994eaf4110ea59f43ab3b6b6503a36 with:
> Fixed editor's background color in landscape mode with dark theme
